plugins/generator-1.20.1/forge-1.20.1/templates/item/item.java.ftl

```diff
@@ -319,10 +336,19 @@ public class ${name}Item extends Item {
 	if (player.getAbilities().instabuild || stack != ItemStack.EMPTY) {
 		<#assign projectileClass = generator.map(projectile, "projectiles", 0)>
 		<#if projectile.startsWith("CUSTOM:")>
-			${projectileClass} projectile = ${projectileClass}.shoot(world, entity, world.getRandom());
+			<#if !data.chargesPower>
+				${projectileClass} projectile = ${projectileClass}.shoot(world, entity, world.getRandom());
+			<#else>
+				<#assign projectileElement = w.getWorkspace().getModElementByName(projectile?replace("CUSTOM:", "")).getGeneratableElement()>

```

Comment:
It would be cleaner (and more performant than loading GE) to add another method overload in projectile.java.ftl